python/pyproject.toml

```diff
@@ -0,0 +1,145 @@
+# =============================================================================
+# PACKAGING: cucumber-tag-expressions
+# =============================================================================
+# SEE: https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html
+# SEE: https://pypi.org/classifiers/
+# MAYBE: requires = ["setuptools", "setuptools-scm"]
+
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+
+[project]
+name = "cucumber-tag-expressions"
+dynamic = ["version"]
+description = "Provides a tag-expression parser and evaluation logic for cucumber/behave"
+readme = "README.rst"
+authors = [
+    { name="Jens Engel", email="jenisys@users.noreply.github.com" },
+]
+dependencies = [
+    "enum34; python_version < '3.4'",
+]
+keywords= ["BDD", "testing", "tag-expressions", "cucumber", "behave"]
+# source-includes = [
+#    "tests/",
+#    "py.requirements/",
+#    ".bumbversion.cfg",
+#    ".coveragerc",
+#    ".editorconfig",
+#    ".envrc",
+#    ".pylintrc",
+#    "LICENSE",
+#    "Makefile",
+#    "**/*.cfg",
+#    "**/*.in",
+#    "**/*.ini",
+#    "**/*.mk",
+#    "**/*.toml",
+#    "**/*.yaml",
+#    "**/*.yml",
+#    "**/*.rst",
+#    "**/*.txt",
+# ]
+requires-python = ">=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*,!=3.5.*"
+# requires-python = ">=3.7"
+license = { text = "MIT"}
```

Comment:
this can be just one of the classifiers, no need to be in a `license` key anymore